routes/serviceInfo.js

```diff
@@ -11,7 +11,8 @@ module.exports = (req, res) => {
         else {
           res.render('serviceInfo', {
             service: service,
-            latlng: coordinates
+            latlng: coordinates,
+            back: req.headers.referer
```

Comment:
req.headers.referer is the URL of the page before the page that the client is currently on. We send this to handlebars, and use the URL as the `href` in the anchor link of the back button.